keps/2724-topology-aware-scheduling/README.md

```diff
@@ -430,6 +430,11 @@ const (
   // at the highest topology level, then it gets admitted as distributed
   // among multiple topology domains.
   PodSetPreferredTopologyAnnotation = "kueue.x-k8s.io/podset-preferred-topology"
+
+  // PodSetUnconstrainedTopologyAnnotation indicates that a PodSet does not have any topology requirements.
+  // Kueue admits the PodSet if there's enough free capacity available.
+  // Recommended for PodSets that don't require pod-to-pod communication
+  PodSetUnconstrainedTopologyAnnotation = "kueue.x-k8s.io/podset-unconstrained-topology"
```

Comment:
What are the differences between this annotation and one without any annotations?

Reply:
Additionally, what is expected value? Bool?

Reply:
Yes, bool is expected value. The difference between this annotation and no annotation is that with TAS we mitigate the risk of admitting a job when capacity is fragmented. Imagine you have 2 GPUs scattered across 2 nodes, and you want to schedule a single pod that requires. Without TAS (and this annotation) a job would be falsely admitted even it cannot run.

A natural question may appear: why not setting just `preferred:hostname`? The difference between those two is that with preferred TAS try to pack pods as tighly as possible. While it's valuable for workloads that require heave pod-to-pod communication, that's not always the case, and a user might prefer to place pods on nodes scattered across the whole datacenter e.g. to reduce fragmentation

Reply:
Added clarification

Reply:
Thank you. I added a clarified question to another thread.